login.php

```diff
@@ -74,24 +79,24 @@
 		
 		<div class="contact_container3">
 			<h2>Login</h2><br>
-			<form action="connect3.php" method="post" target="_blank">
+			<form method="post" method="post" target="_blank" id="login_user">

```

Comment:
form code changed so that the login actually functions - id given so that the session functions properly across the website